### DISCUSSION TITLE
Debian default grub theme

### DISCUSSION THREAD

**Discussion by robertocannito** [SpiralLinux/SpiralLinux-project#280] [Q&A]
Hi, I'm running in sid, how is possible change to debian default boot grub default image or theme?
I know in /boot/grub there's a file splash.png but there's a way that debian change me automatically this image every time they change? 

I hope that all is clear  :)

**Comment by geckolinux** [comment]
Hi @robertocannito , the easiest way to change that is with the GRUB Customizer program that SpiralLinux includes by default.

**Comment by robertocannito** [comment]
Sam, there is another way more easy, just cancel splash.png and update grub
If I can I avoid to use grubcustomizer.

**Reply by geckolinux** [reply]
This would be the manual process:
https://www.thegeekstuff.com/2012/10/grub-splash-image/

**Comment by literalmock** [comment]
Hope you're doing well i had thought that you create a discord server it would be supercool for discussions and and issues we can solve that live also as you said you are holding bookworm update for debian VM support repo to be active it's absolutely fine but till you can post the beta release their for the users who wanna try that can try on the actual machine. 
Looking forward to your thoughts on this

**Comment by methanoid** [comment]
Please NO to Discord.. it just becomes another place to split the information available. What's wrong with here?
